e2e/api/steps.js

```diff
@@ -52,6 +52,8 @@ module.exports = {
 
     await assertCorrectEventsAreAvailableToUser(config.solicitorUser, 'AWAITING_CASE_NOTIFICATION');
     await assertCorrectEventsAreAvailableToUser(config.defendantSolicitorUser, 'AWAITING_CASE_NOTIFICATION');
+    await assertCallbackError('Court', data[eventName].invalid.Court.courtLocation.applicantPreferredCourt,
```

Comment:
this method name is now incorrect, we are not asserting a callback error. Better way to do this could be an entirely new method that just calls assertCallbackError inside? Or maybe renaming existing method, thoughts?

Reply:
Could we rename the method to just assertError?

Reply:
I'd say it's better to have a separate method for that to have clear separation between callback and data store errors. We could even use it later for testing other ccd validation bits that are not covered now, like min/max values.